packages/excalidraw/actions/actionFinalize.tsx

```diff
@@ -50,7 +50,6 @@ export const actionFinalize = register({
             ...appState,
             cursorButton: "up",
             editingLinearElement: null,
-            selectedLinearElement: null,
```

Comment:
is this intentional?

Reply:
Yes, and this change was in this [commit](https://github.com/excalidraw/excalidraw/pull/8340/commits/a8e978731e73d71869105e12f23dda4d6b28902a). Such that, after exiting the point editing mode of a linear element, the linear element is still selected with the little control points shown. IIRC, we talked briefly about it on one of our calls and maintaining the little control points was the conclusion that we reached (for consistency reason).